src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/rightcurly/InputRightCurlyTestAloneOrSingleline.java

```diff
@@ -10,6 +10,9 @@
 
 package com.puppycrawl.tools.checkstyle.checks.blocks.rightcurly;
 
+import java.io.BufferedReader;
+import java.io.IOException;
+
 public class InputRightCurlyTestAloneOrSingleline {
```

Comment:
Move new test to a whole new file. We don't like modifying existing tests and causing line differences.

Reply:
done